pmda/density.py

```diff
@@ -266,8 +269,9 @@ def _prepare(self):
             smin = np.min(coord, axis=0)
             smax = np.max(coord, axis=0)
             # Overwrite smin/smax with user defined values
-            smin, smax = _set_user_grid(self._gridcenter, self._xdim,
-                                        self._ydim, self._zdim, smin, smax)
+            smin, smax = set_user_grid(
```

Comment:
When 2.0.0 is out, we need test enviroments for both MDAnalysis 1.0.0 and 2.0.0.

Reply:
Yes, true. Something to keep in mind.

We also have to consider for how long we actually want to support MDA 1.x.